app/Round.js

```diff
@@ -0,0 +1,27 @@
+import {Match} from './Match.js';
+export class Round{
+  constructor(numberofMatches){
+    this._numberofMatches=numberofMatches;
+    this._Matches=[];
+  }
+  get numberofMatches() {
+      return this._numberofMatches;
+  }
+  set numberofMatches(numberofMatches) {
+      this._numberofMatches = numberofMatches;
+  }
+  get Matches() {
+      return this._Matches;
+  }
+  set Matches(Matches) {
+      this._Matches = Matches;
+  }
+  generateMatches(){
+    for(var i=0;i<this.numberofMatches;i++){
+      this._Matches.push(new Match());
+    }
+  }
+  addMatch(Player1,Player2){
+    this._Matches.push(new Match(Player1,Player2));
+  }
```

Comment:
These 2 methods don't enforce a correct way of using the class. It's very easy to make it enter on inconsistent state. Even this usage:  
`round.numberOfMatches = 4;  
round.generateMatches();`  
Is more error prone than  
`round.generateMatches(4);`